ichnaea/content/tests/test_views.py

```diff
@@ -87,36 +87,40 @@ def test_stats(self, session, session_tracker, views):
 
 
 class TestFunctionalContent(object):
-    def test_content(self, app, session_tracker, metricsmock):
-        app.get("/", status=200)
-        app.get("/apple-touch-icon-precomposed.png", status=200)
-        app.get("/api", status=200)
-        app.get("/contact", status=200)
-        app.get("/favicon.ico", status=200)
-        app.get("/map", status=200)
-        app.get("/nobody-is-home", status=404)
-        app.get("/optout", status=200)
-        app.get("/privacy", status=200)
-        app.get("/robots.txt", status=200)
-        app.get("/static/css/images/icons-000000@2x.png", status=200)
-        app.get("/terms", status=200)
-        session_tracker(0)
-        app.get("/stats/regions", status=200)
-        session_tracker(1)
-        app.get("/stats", status=200)
-        session_tracker(8)
-        assert metricsmock.has_record(
-            "incr", "request", value=1, tags=["path:", "method:get", "status:200"]
-        )
-        assert metricsmock.has_record(
-            "incr", "request", value=1, tags=["path:map", "method:get", "status:200"]
-        )
-        assert metricsmock.has_record(
-            "timing", "request.timing", tags=["path:", "method:get"]
-        )
-        assert metricsmock.has_record(
-            "timing", "request.timing", tags=["path:map", "method:get"]
-        )
+    @pytest.mark.parametrize(
+        "path,status,metric_path,db_calls",
+        (
+            ("/", 200, "__homepage__", 0),
+            ("/apple-touch-icon-precomposed.png", 200, None, 0),
+            ("/api", 200, "api", 0),
+            ("/contact", 200, "contact", 0),
+            ("/favicon.ico", 200, None, 0),
+            ("/map", 200, "map", 0),
+            ("/nobody-is-home", 404, None, 0),
+            ("/optout", 200, "optout", 0),
+            ("/privacy", 200, "privacy", 0),
+            ("/robots.txt", 200, None, 0),
+            ("/static/css/images/icons-000000@2x.png", 200, None, 0),
```

Comment:
I think these static assets are why there is a special case for the ``@`` symbol. There are some assets like https://location.stage.mozaws.net/static/images/mls-logo@2x.png that are used in high-resolution displays like a MacBook.